drivers/i2c/i2c_ll_stm32_v2.c

```diff
@@ -135,6 +136,25 @@ void stm32_i2c_error_isr(void *arg)
 
 	k_sem_give(&data->device_sync_sem);
 }
+#endif
+
+#ifdef CONFIG_I2C_STM32_COMBINED_INTERRUPT
+void stm32_i2c_combined_isr(void *arg)
```

Comment:
The combined isr doesn't takes into account the I2C error interrupts. I found out that i2c v2 driver doesn't enable I2C error interrupts (I thought it did), even if it checks for them in error isr. We should fix this probably in another PR.

Reply:
OK